src/switch.js

```diff
@@ -0,0 +1,271 @@
+/**
+ * @license
+ * Copyright 2022 MIT
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+/**
+ * @fileoverview Multiple selection svg switch.
+ */
+
+import {WorkspaceMultiSelect} from '../src/index';
+import * as Blockly from 'blockly/core';
+
+/**
+ * Icon path of the multi select controls when enabled.
+ * @type {string}
+ * @const
+ */
+const ENABLED_IMG = 'media/select.svg';
```

Comment:
If you want other developers to use this, it might be nice to make the images configurable via the config passed to `WorkspaceMultiSelect`.

Reply:
Okay.